src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -162,3 +168,10 @@ function undo_steps_for_delete_remote_branch {
   echo "push_branch $branch_to_delete"
 }
 
+
+function undo_steps_for_delete_remote_only_branch {
+  local branch_to_delete="$1"
+  local remote_sha="$(git log origin/"$branch_to_delete" | head -1 | cut -d ' ' -f 2)"
+  echo "create_branch $branch_to_delete $remote_sha"
+  echo "push_branch $branch_to_delete"
+}
```

Comment:
I think this should just be joined with undo_steps_for_delete_remote_branch. 
It can just check if a local branch exists by that name and if not it creates it